util/src/main/scala/org/ensime/util/file.scala

```diff
@@ -115,6 +115,9 @@ package object file {
         case t: Throwable => file.getAbsoluteFile
       }
 
+    def descendsFrom(parent: File): Boolean =
+      file.getParent.startsWith(parent.getAbsolutePath)
```

Comment:
I don't think you need to do any conversions if you pass the right higher level object to your filter.